.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Debezium Incubator
+
+on: [push, pull_request]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Pre Install Setup
+        run: |
+          sudo service mysql-5.6 stop || sudo service mysql stop
+          sudo /etc/init.d/postgresql stop
+      - name: Install and Setup Maven
+        run: |
+          wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip
+          unzip -qq apache-maven-3.6.3-bin.zip
+          export M2_HOME=$PWD/apache-maven-3.6.3
+          export PATH=$M2_HOME/bin:$PATH
+          mvn -version
+      - name: Cache local Maven repository
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Setup Debezium Parent
+        run: git clone https://github.com/debezium/debezium.git $PWD/core-repo && cd $PWD/core-repo && mvn -B clean install -DskipTests -DskipITs -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn && cd $OLDPWD
+      - name: Setup Debezium Incubator
+        run: mvn -B clean install -Passembly -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
I'm curious for consistency if we shouldn't look at creating multiple workflows much like we did in the core repository where we can control which connectors get built depending on the changed files of a push or pull request?  We don't build Oracle on CI presently, so we'd really only need to focus on a cassandra and db2 workflow presently.  wdyt?

Reply:
That's a better one I guess, will be adding separate workflows for `cassandra` and `db2`.